### PR TITLE
feat: update causes subscription payment method

### PR DESCRIPTION
**Pull request by eddieferrer**
GD-194

See ticket for more details
Screen Shot 2021-12-21 at 12 40 02 PM
Screen Shot 2021-12-21 at 12 39 47 PM

### CODE REVIEW

File: src/components/Causes/CausesDropInPaymentWrapper.vue [hunk 1]
```diff
@@ -110,10 +112,48 @@ export default {
 				});
 		},
 		doBraintreeCausesSubscription(nonce, deviceData, paymentType) {
-			/** ! TODO add if (this.action === 'Update') logic here similar to
-			* src/components/MonthlyGood/MonthlyGoodDropInPaymentWrapper.vue
-			* for editing a cause subscription GD-155
-			*/
+			if (this.action === 'Update') {
+				console.log(nonce, deviceData, paymentType);
```
Comment (mcstover) [inline src/components/Causes/CausesDropInPaymentWrapper.vue:116]: bump

**Review by mcstover — APPROVED** [review]
Full feature parity!

File: src/components/Causes/CausesDropInPaymentWrapper.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/Causes/CausesDropInPaymentWrapper.vue:116]: 🤦  Did we not have a lint warning for this at one time?

Comment (mcstover) [reply src/components/Causes/CausesDropInPaymentWrapper.vue]: I thought we had a warning but it def doesn't block anything...I don't see it either at this point in the builds

**Pull request merged**